push_dialog_base.ui

```diff
@@ -6,101 +6,284 @@
    <rect>
```

Comment:
I did this, but when I do that, update_strings.sh creates a .ts file that has the wrong path (ie, still foo.ui  instead of ui/foo.ui) even though it's detected by its full (relative) path in the beginning. I wonder if that is a problem.